### PR TITLE
Refactor webui controller

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
[P9: (8) Refactor Webui::WebuiController](https://trello.com/c/hKKh5pcb/698-p9-8-refactor-webuiwebuicontroller)

File: src/api/app/helpers/webui/webui_helper.rb [hunk 1]
```diff
@@ -122,7 +122,7 @@ def tab(id, text, opts)
     opts[:package] = @package.to_s if @package
     opts[:project] = @project.to_s if @project
     link_opts = { id: "tab-#{id}" }
-    if (@current_action.to_s == opts[:action].to_s && @current_controller.to_s == opts[:controller].to_s) || opts[:selected]
+    if (action_name == opts[:action].to_s && controller_name == opts[:controller].to_s) || opts[:selected]
```
Comment (Ana06) [inline src/api/app/helpers/webui/webui_helper.rb]: the `@current_action` variable is used in the tests: https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/helpers/webui/webui_helper_spec.rb#L222

Test need to be fixed :wink:

Comment (bgeuken) [reply src/api/app/helpers/webui/webui_helper.rb]: @Ana06 Done :wink:

**Comment by Ana06** [comment]
The test coverage decreases... should we merge this or should we add more test first? :thinking:

**Comment by bgeuken** [comment]
According to [codecov](https://codecov.io/gh/openSUSE/open-build-service/compare/ce5c23c270e6fd9b27ed83fe4477559551d3656b...3783a0e4f75746bbb1c849d0b5c6b7dca388bd0c/changes) it's just one line and it doesn't seem this was tested intentionally before.

So I think it's fine to merge this

**Comment by bgeuken** [comment]
[P3: (2) Find out all policies we need for ec2 uploads](https://trello.com/c/zfxP42mC/681-p3-2-find-out-all-policies-we-need-for-ec2-uploads)

**Pull request merged**